File: src/main/java/com/github/brankale/dto/RatingDto.java
package com.github.brankale.dto;

public record RatingDto(
        String type,
        float ratingValue,
        int reviewCount
) {
    @Override
    public String toString() {
        return "RatingDto{" +
                "type='" + type + '\'' +
                ", ratingValue=" + ratingValue +
                ", reviewCount=" + reviewCount +
                '}';
    }
}

File: src/main/kotlin/com/github/brankale/parsers/game_page/dto/RatingDto.kt
package com.github.brankale.parsers.game_page.dto

import com.google.gson.annotations.SerializedName

data class RatingDto(
        @SerializedName("@type") val type: String,
        @SerializedName("ratingValue") val ratingValue: Float,
        @SerializedName("reviewCount") val reviewCount: Int
)
